static/themes/SSF/style.css

```diff
@@ -46,3 +46,9 @@
     background-image: url(../../img/sunflower_spin_throbber.gif);
     background-repeat: no-repeat;
 }
+
+/* Comments */
+
+#comment-form {
+    width: 100%;
```

Comment:
Not sure about eden tests...
but if you want something to work on:
try this  http://sunflower.sahanafoundation.org/eden/project/task/389 ?